### PR TITLE
Added X507UF

### PR DESCRIPTION
**Pull request by vaibhav-hexa**
**Description**
Added X507UF model along with the corresponding changes.

**Additional data**
Related to issues:
- https://github.com/dominiksalvet/asus-fan-control/issues/73
- https://github.com/dominiksalvet/asus-fan-control/issues/68

### CODE REVIEW

File: data/models [hunk 1]
```diff
@@ -13,3 +13,4 @@ VivoBookS15X530UA|1335|55 57 59 61 63 65 67 69
 X580VD|1335 1431|45 50 55 60 65 70 75 80
 TUFGamingFX505GMFX505GM|1335 1431|45 50 55 60 65 70 75 80
 VivoBook15ASUSLaptopX542UF|1335|40 45 50 55 59 63 65 67
+VivoBook15ASUSLaptopX507UF|1335|55 60 62 65 68 72 76 80
```
Comment (dominiksalvet) [inline data/models:16]: In the #73 you mentioned `55 60 65 70 75 80 85 90` as `get-temps`, you have different temps here. Please clarify, which ones you want to use as default for your laptop model.

**Review by dominiksalvet — COMMENTED** [review]
Is this https://www.asus.com/Laptops/For-Students/All-series/ASUS-X507 your laptop? If so, we will present it as ASUS X507UF since ASUS does so (even though `dmidecode` tells otherwise). I am trying to do it with every single model added.

Please, answer the question about the temperatures (or push changes) and we will be ready to merge. :rocket:

**Comment by vaibhav-hexa** [comment]
> Is this https://www.asus.com/Laptops/For-Students/All-series/ASUS-X507 your laptop? If so, we will present it as ASUS X507UF since ASUS does so (even though `dmidecode` tells otherwise). I am trying to do it with every single model added.

Yes, that's my laptop.
 
> Please, answer the question about the temperatures (or push changes) and we will be ready to merge.

Sorry about the confusion. Yes, `55 60 65 70 75 80 85 90` is the correct temp-range. 
It is the exact same range obtained from `afc-scout` and based on my usage that's the appropriate temp-range for this laptop.

image

**Review by dominiksalvet — APPROVED** [review]
LGTM, merging soon.

Thank you so much for all your contributions! 👏👏 AFC has become a bit better. ❤️

Closes #73

File: readme.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Asus-fan-control solves the problems above and even more. **Tested devices** wit
 | ASUS          | ASUS TUF Gaming | ASUS VivoBook       | ASUS ZenBook       | ASUSPRO      | ROG            |
 |---------------|-----------------|---------------------|--------------------|--------------|----------------|
 | [FX553VD][12] | [FX505GM][15]   | [15 X510UA][4]      | [Flip UX360UAK][6] | [B9440UA][1] | [G752VL][2]    |
-|               |                 | [15 X512FA][5]      | [UX410UA][7]       |              | [GL502VSK][10] |
+| [X507UF][17]  |                 | [15 X512FA][5]      | [UX410UA][7]       |              | [GL502VSK][10] |
```
Comment (FadeMind) [inline readme.md:20]: @dominiksalvet this is VivoBook too

Comment (vaibhav-hexa) [reply readme.md:20]: Yes, it's a Vivobook 15 series laptop. But, as Dominik mentioned in another issue, (https://www.asus.com/Laptops/For-Home/All-series/ASUS-X507), it's not Vivobook anymore.

Comment (dominiksalvet) [reply readme.md:20]: I will always respect official ASUS naming at the time of adding untested devices. And this case is no exception. Nevertheless, I must admit that even where I live some local distributors label it as a VivoBook (just as `dmidecode` reported). That will not change the presented name, though. Does this explanation make sense?

**Pull request merged**